examples/create_ingress.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kubernetes import client, config
+
+
+def main():
+    # Fetching and loading Kubernetes Information
+    config.load_kube_config()
+    # For incluster details
+    # config.incluster_kube_config()
+
+    extension = client.ExtensionsV1beta1Api()
```

Comment:
[the official doc](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource) recommends using `networking.k8s.io/v1beta1` api instead of extensions v1beta1